dao/src/main/java/org/thingsboard/server/dao/sqlts/insert/sql/SqlPartitioningRepository.java

```diff
@@ -64,20 +61,16 @@ public void createPartitionIfNotExists(String table, long entityTs, long partiti
             partitionCreationLock.lock();
             try {
                 if (partitions.containsKey(partitionStartTs)) return;
-                log.trace("Saving partition: {}", partition);
+                log.info("Saving partition {}-{} for table {}", partition.getStart(), partition.getEnd(), table);
                 save(partition);
                 log.trace("Adding partition to map: {}", partition);
                 partitions.put(partition.getStart(), partition);
-            } catch (RuntimeException e) {
-                log.trace("Error occurred during partition save:", e);
-                String msg = ExceptionUtils.getRootCauseMessage(e);
-                if (msg.contains("would overlap partition")) {
-                    log.warn("Couldn't save {} partition for {}, data will be saved to the default partition. SQL error: {}",
-                            partition.getPartitionDate(), table, msg);
+            } catch (Exception e) {
+                String error = ExceptionUtils.getRootCauseMessage(e);
+                if (StringUtils.containsAny(error, "would overlap partition", "already exists")) {
                     partitions.put(partition.getStart(), partition);
-                } else {
-                    throw e;
                 }
+                log.warn("Couldn't save partition {}-{} for table {}: {}", partition.getStart(), partition.getEnd(), table, error);
```

Comment:
We already know the root cause, so let's log this line only in the "else" block:

```
if (StringUtils.containsAny(error, "would overlap partition", "already exists")) {
    partitions.put(partition.getStart(), partition);
 } else {
    log.warn("Couldn't save partition {}-{} for table {}: {}", partition.getStart(), partition.getEnd(), table, error);                
 }

```